control-spec.txt

```diff
@@ -2098,6 +2107,19 @@
    actual reason given by the remote OR for closing the circuit. Clients MUST
    accept reasons not listed above.  Reasons are as listed in tor-spec.txt.
 
+   The "UNUSABLE_FOR_NEW_CONNS" field is provided only in versions 0.4.X and
+   later. UNUSABLE_FOR_NEW_CONNS is a boolean digit set to 1 if no new
+   connection can be opened on the circuit or 0 if further connections can be
+   opened.
+
+   The "IDLE_TIMEOUT" field is provided only in versions 0.4.X and later.
```

Comment:
We should document briefly that this value is set when the circuit is constructed, and does not "count down".

Reply:
Resolved by: "Please note that this field is not a countdown."